lib/PasswordPolicyConfig.php

```diff
@@ -181,6 +182,57 @@ public function setEnforceHaveIBeenPwned(bool $enforceHaveIBeenPwned) {
 		$this->config->setAppValue('password_policy', 'enforceHaveIBeenPwned', $enforceHaveIBeenPwned ? '1' : '0');
 	}
 
+	/**
+	 * number of words in passphrase
+	 *
+	 * @return int
+	 */
+	public function getWordCount(): int {
+		return $this->config->getAppValue(
+			'password_policy',
+			'wordCount',
+			4
+		);
+	}
+
+	/**
+	 * number of words in passphrase
+	 *
+	 * @param int $numWords
+	 */
+	public function setWordCount(int $numWords) {
+		$this->config->setAppValue(
+			'password_policy',
+			'wordCount',
+			$numWords);
+	}
+
+
+	/**
+	 * generate passphrases
+	 *
+	 * @return bool
+	 */
+	public function getGeneratePassphrases(): bool {
+		return $this->config->getAppValue(
+			'password_policy',
+			'genPhrases',
+			'1'
+		) === '1';
+	}
+
+	/**
+	 * generate passphrases
+	 *
+	 * @param bool $genPhrases
```

Comment:
You can remove this annotation because the var of method already is typed.

Other point is that the method already have a clear name and I think that isn't necessary to have a description with the same name of method. I think that will be best remove all of this docblock.
```suggestion
	 * generate passphrases
```